packages/cli/lib/discoverManifests.js

```diff
@@ -9,10 +8,12 @@ import { packageDirectory } from 'pkg-dir'
 import { findUp } from 'find-up'
 
 const packagePattern = /^barnard59-(.+)$/
-const require = module.createRequire(import.meta.url)
 
-export default async function * () {
-  const packages = (await getInstalledPackages()).filter(hasManifest)
+export default async function * ({ basePath = import.meta.url, all = false } = {}) {
+  const require = module.createRequire(basePath)
+  const hasManifest = canRequireManifest.bind(null, require)
```

Comment:
The `basePath` cannot be the module URL because in global context, `barnard59-*` packages cannot be resolved from the CLI package. Thus, when loading the pipeline, its path will be used as base for resolution